front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -316,27 +317,33 @@ export function AgentMessage({
     );
   }
 
+  const { configuration: agentConfiguration } = agentMessageToRender;
+
   return (
     <ConversationMessage
       owner={owner}
       user={user}
       conversationId={conversationId}
       messageId={agentMessageToRender.sId}
-      pictureUrl={agentMessageToRender.configuration.pictureUrl}
-      name={`@${agentMessageToRender.configuration.name}`}
+      pictureUrl={agentConfiguration.pictureUrl}
+      name={`@${agentConfiguration.name}`}
       buttons={buttons}
       avatarBusy={agentMessageToRender.status === "created"}
       reactions={reactions}
       enableEmojis={true}
       renderName={() => {
         return isDevelopmentOrDustWorkspace(owner) ? (
-          <div className="text-sm font-medium">
-            {AssitantDetailViewLink(agentMessageToRender.configuration)}
+          <div className="flex flex-row gap-2">
+            <div className="text-sm font-medium">
+              {AssitantDetailViewLink(agentConfiguration)}
+            </div>
+            <AssistantQuickEditionMenu
```

Comment:
nit: AssistantEditionMenu 
Also, :pray: will reuse for 3265 rn

Reply:
okok